client/state/happiness-engineers/selectors.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+
+function getHappinessEngineersState( state ) {
+	return state.happinessengineers;
+}
```

Comment:
Personally, I'd rather do without this extra level of indirection and use `state.happinessengineers` here, but that's up to your taste :-)